### DISCUSSION TITLE
How can I update my current best-model with new data?

### DISCUSSION THREAD

**Discussion by JavierAEM94** [explosion/spaCy#13428] [Help: Coding & Implementations]
I have created a custom NER model with some training data, but I need to add new data to improve my current model. Is there a way to re-train my model with new data?

**Comment by svlandeg** [comment]
This topic has been discussed at length in previous threads - you can search for them on Github / Google, or check out our [FAQ](https://github.com/explosion/spaCy/discussions/8226) for related pointers. Let us know if you run into any specific issues!
